src/node/delay.rs

```diff
@@ -29,6 +29,41 @@ impl Default for DelayOptions {
 ///
 /// The current implementation does not allow for zero delay. The minimum delay is one render
 /// quantum (e.g. ~2.9ms at 44.1kHz).
+///
```

Comment:
I really like the style of documenting each node in detail. I will update my panner-node PR in the same fashion, and let's improve over time the other node's docs

Reply:
Ok cool I'll also continue on that vein too then